bloopark-addons/bp_food/data/data.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<odoo>
+    <data>
+        <record id="seq_food_table" model="ir.sequence">
+            <field name="name">Table No.</field>
+            <field name="code">food.tables.sequence</field>
+            <field name="prefix"></field>
+            <field name="padding">1</field>
+            <field name="company_id" eval="False"/>
+        </record>
+
+        <record id="seq_food_order" model="ir.sequence">
+            <field name="name">Order No.</field>
+            <field name="code">food.order.sequence</field>
```

Comment:
'food.order.sequence' shouldn't be 'food.order'? Is it working like this? It shouldn't.